cpp/log/database_test.cc

```diff
@@ -47,7 +47,12 @@ typedef testing::Types<FileDB<cert_trans::LoggedCertificate>,
 
 typedef Database<cert_trans::LoggedCertificate> DB;
 
+template <class T>
+class DBTestDeathTest : public DBTest<T> {
```

Comment:
That doesn't seem to work. Possibly because TYPED_TEST_CASE is a macro? It produces lots of:

In file included from ../gmock-1.7.0/gtest/include/gtest/gtest.h:66:0,
                 from cpp/log/database_test.cc:2:
../gmock-1.7.0/gtest/include/gtest/gtest-typed-test.h:172:24: error: expected template-name before ‘<’ token
       : public CaseName<gtest_TypeParam_> { \
                        ^
cpp/log/database_test.cc:369:1: note: in expansion of macro ‘TYPED_TEST’
 TYPED_TEST(DBTestDeathTest, CannotOverwriteNodeId) {
 ^